docs/current_docs/api/snippets/default-paths/php/src/MyModule.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DaggerModule;
+
+use Dagger\Attribute\{DaggerFunction, DaggerObject, DefaultPath, ReturnsListOfType};
+use Dagger\{File, Directory};
+
+#[DaggerObject]
+class MyModule
+{
+    #[DaggerFunction]
+    #[ReturnsListOfType('string')]
+    public function readDir(
+        #[DefaultPath('.')]
+        Directory $source,
+    ): array {
+        return $source->entries();
+    }
+
+    #[DaggerFunction]
+    public function readFile(
```

Comment:
I think we can dispense with the readFile() function, it was removed from the other languages in the last update of this page.